<launch>
    <rosparam command="load" file="$(find cart_pole_3d_training_pkg)/config/cart_pole_3d.yaml" />
    <!-- Launch the training system -->
    <node pkg="cart_pole_3d_training_pkg" name="cart_pole_3d_gym" type="training_ddqn.py" output="screen"/>
</launch>
